apps/communications/dialer/test/marionette/call_log_test.js

```diff
@@ -1,78 +0,0 @@
-'use strict';
```

Comment:
oh! I got confused..thought this test was one of my testing tests...:S will add it :)